lldb/source/Target/Target.cpp

```diff
@@ -5066,3 +5066,17 @@ llvm::json::Value
 Target::ReportStatistics(const lldb_private::StatisticsOptions &options) {
   return m_stats.ToJSON(*this, options);
 }
+
+bool Target::SectionLoadListIsEmpty() const {
+  return const_cast<Target *>(this)->GetSectionLoadList().IsEmpty();
```

Comment:
To avoid the `const_cast`, can we have a `const` `GetSectionLoadList` overload? `SectionLoadHistory::IsEmpty()` is already `const`.

Reply:
We actually can't because `SectionLoadHistory::GetCurrentSectionLoadList()` needs to be able to create an empty section load for the special stop ID of `eStopIDNow` if the section load list is empty. I don't thing we want to change section load list to have the load list be mutable. Let me know if you disagree or have another solution?

Reply:
I found a way to fix this. We don't need the  this function to be `const`